include/ultra64/gbi.h

```diff
@@ -1054,6 +1056,27 @@ typedef struct {
 	unsigned char	v[3];
 } Tri;
 
+typedef long int Mtx_t[4][4];
+typedef union {
+    Mtx_t m;
+    struct {
+        u16 intPart[4][4];
+        u16 fracPart[4][4];
+    };
+    long long int force_structure_alignment;
+} Mtx; // size = 0x40
+
+typedef float MtxF_t[4][4];
+typedef union {
+    MtxF_t mf;
+    struct {
+        float xx, yx, zx, wx,
+              xy, yy, zy, wy,
+              xz, yz, zz, wz,
+              xw, yw, zw, ww;
+    };
+} MtxF; // size = 0x40
```

Comment:
This is probably better off in `z64math.h`, it definitely shouldn't be in here at least